docs/labwc-actions.5.scd

```diff
@@ -50,6 +54,17 @@ Actions are used in menus and keyboard/mouse bindings.
 	top or bottom tell how much to resize on that edge of window,
 	positive values grow window, negative shrink window.
 
+*<action name="GrowToEdge" direction="value" />*
+	Resize window to fill the space between its edge and any other
+	window edge.
+
+	*direction* [left|up|right|down] Direction in which to grow.
```

Comment:
I'd still argue that at least `GrowToEdge` should additionally include the optional `snap` / `ignoreWindow` argument so users are able to grow a view until it hits the next screen edge, regardless of any other windows in between. I am not sure right now if the same would make sense for `ShrinkToEdge`. However I'd be fine to delay this to the future as an existing config wouldn't break by adding that argument.

Reply:
Later.